typescript/packages/well-log-viewer/src/SyncLogViewer.stories.tsx

```diff
@@ -159,7 +162,7 @@ function fillInfo(controller: WellLogController | undefined) {
 }
 
 // @ts-expect-error TS7006
```

Comment:
Suggest removing the suppression now that it's fixed.

Reply:
// @ts-expect-error TS7006 
is removed